packages/components/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,194 @@
+                            <Dropdown.Trigger data-test-id="overflow-items-dropdown-trigger">
+                                <Button emphasis="default" aspect="square" size="small">
+                                    <IconDotsHorizontal size={16} />
+                                </Button>
+                            </Dropdown.Trigger>
+                        )}
+                        <Dropdown.Content align="end" data-test-id="overflow-items-dropdown-content">
+                            {triggersOutOfView.map((trigger) => (
+                                <Dropdown.Item
+                                    disabled={trigger.disabled}
+                                    onSelect={() => {
+                                        setActiveTab(trigger.value);
+                                    }}
+                                    key={trigger.value}
+                                >
+                                    {trigger.element}
+                                </Dropdown.Item>
+                            ))}
+                        </Dropdown.Content>
+                    </Dropdown.Root>
+                    <span
+                        data-test-id="active-tab-indicator"
+                        ref={activeIndicatorRef}
+                        className={styles.activeIndicator}
+                    />
+                </div>
+                {children}
+            </RadixTabs.Root>
+        </TabTriggerContext.Provider>
+    );
+};
+TabsRoot.displayName = 'Tabs.Root';
+
+type TabsTabProps = {
+    children: ReactNode;
+    value: string;
+    disabled?: boolean;
+};
+
+export const TabsTab = ({ children, value, disabled }: TabsTabProps) => (
+    <TabConfigContext.Provider value={{ value, disabled: !!disabled }}>{children}</TabConfigContext.Provider>
+);
+TabsTab.displayName = 'Tabs.Tab';
+
+type TabsTriggerProps = {
+    children: ReactNode;
+};
+
+export const TabsTrigger = ({ children, ...props }: TabsTriggerProps, ref: ForwardedRef<HTMLButtonElement>) => {
+    const { value, disabled } = useContext(TabConfigContext);
+
+    const { addTrigger } = useContext(TabTriggerContext);
+
+    const localRef = useRef<HTMLButtonElement>(null);
+
+    useEffect(() => {
+        addTrigger({
+            ref: localRef || ref,
+            value: value || '',
+            disabled,
+            element: (
+                <button ref={localRef || ref} {...props} data-bla="ttt">
+                    {children}
+                </button>
+            ),
+        });
+    }, []);
+
+    return null;
+};
+TabsTrigger.displayName = 'Tabs.Trigger';
+
+type TabsContentProps = {
+    children: ReactNode;
+};
+
+export const TabsContent = ({ children, ...itemProps }: TabsContentProps, ref: ForwardedRef<HTMLDivElement>) => {
+    const { value } = useContext(TabConfigContext);
+
+    return (
+        <RadixTabs.Content ref={ref} {...itemProps} className={styles.content} value={value || ''}>
```

Comment:
Prefer `??` to `||` when possible
```suggestion
        <RadixTabs.Content ref={ref} {...itemProps} className={styles.content} value={value ?? ''}>
```

Reply:
fixed